src/clients/ntp-client_proposal.rb

```diff
@@ -1,11 +1,18 @@
 require "yast"
 
+require "y2ntp_client/widgets/sources_table"
+
 module Yast
   # This is used as the general interface between yast2-country
   # (time,timezone) and yast2-ntp-client.
   class NtpClientProposalClient < Client
     include Yast::Logger
 
+    @sources_table = nil
```

Comment:
I would omit this initialization and move it to a method...
```suggestion
    def sources_table
      @sources_table_widget ||= Y2NtpClient::Widgets::SourcesTable.new(NtpClient.GetUsedNtpSources)
    end
```